pyglider/slocum.py

```diff
@@ -884,6 +884,16 @@ def binary_to_timeseries(indir, cachedir, outdir, deploymentyaml, *,
         if sensorname in dbd.parameterNames['sci']:
             _log.debug('Sci sensorname %s', sensorname)
             val = data[nn]
+
+            # interpolate only over those gaps that are smaller than 'maxgap'
+            _t, _ = dbd.get(ncvar[name]['source'])
+
+            try:
+                tg_ind = utils.find_gaps(_t,time,maxgap) 
+                val[tg_ind] = np.nan            
+            except ValueError:
```

Comment:
Did you hit this exception?  Why?  A print statement probably is wrong.  Either warm or log.

Reply:
I never hit the exception, and the print statement should've been removed after testing. I've done that now. Thanks!